go.mod

```diff
@@ -34,6 +35,7 @@ require (
 	github.com/onflow/flow-core-contracts/lib/go/contracts v0.1.0
 	github.com/onflow/flow-go-sdk v0.8.0
 	github.com/onflow/flow-go/crypto v0.9.4
+	github.com/onflow/flow-go/integration v0.0.0-00010101000000-000000000000 // indirect
```

Comment:
this shouldn't be there - we don't want the main `go.mod` to not include `integration` module since integration includes things from the main package.

Reply:
fixed in https://github.com/onflow/flow-go/pull/153/commits/a46157013889b96a219f74cee76e981b7cb690a5